src/ContactForm.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react';
+
+class ContactForm extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      name: '',
+      avatar: '',
+      occupation: '',
+    }
+  }
+
+  handleNameChange(event) {
```

Comment:
I'm struggling greatly with react. To the point that I kind of don't know what questions to ask. I've begun studying more outside of class this week but I'm still very behind as far as understanding. I plan to spend a lot of this next week just trying to get the basics down. This is why I couldn't come up with any of the extensions. Promise I tried! Will continue trying.